planner/core/testdata/integration_suite_out.json

```diff
@@ -2355,7 +2355,7 @@
       {
         "SQL": "explain format = 'verbose' select * from t3 order by a",
         "Plan": [
-          "Sort_4 3.00 45.85 root  test.t3.a",
+          "Sort_4 3.00 26.05 root  test.t3.a",
```

Comment:
We can ignore this change since it is caused by post optimization, and the original plan is:
```
[Sort_4 3.00 45.85 root  test.t3.a]
[└─Projection_6 3.00 31.58 root  test.t3.a, test.t3.b]
[  └─TableReader_8 3.00 11.78 root  data:TableFullScan_7]
[    └─TableFullScan_7 3.00 108.00 cop[tikv] table:t3 keep order:false]
```
After post-optimization, the projection is removed and the plan cost becomes `45.85(original plan cost)` - `31.58-11.78(projection cost)` = `26.05(the new plan cost)`.